students/andrew_garcia/lesson_06/mailroom_test.py

```diff
@@ -0,0 +1,66 @@
+'''
+Andrew Garcia
+Mailroom Tests
+8/7/19
+'''
+
+import Mailroom4 as mr
+import os.path
+
+def test_sorting_donors():
+    """Tests sorting donors by highest donation amount"""
+    assert mr.sorting_donors() == [['Bryan Fultan', 1, 1236.28, 1236.28], ['Jayson Black', 3, 584.04, 194.67999999999998], ['Danica Dolores', 3, 450.15, 150.04999999999998], ['Roy Max', 2, 365.6, 182.8], ['Skylar Odell', 2, 319.52, 159.76]]
+
+def test_thank_all():
+    """Tests file creation for donors"""
+    mr.thank_all()
+    assert os.path.exists('Bryan Fultan.txt')
+    assert os.path.exists('Danica Dolores.txt')
+    assert os.path.exists('Jayson Black.txt')
+    assert os.path.exists('Roy Max.txt')
+    assert os.path.exists('Skylar Odell.txt')
+
+def test_thank_all_files():
+    """Tests content in a file created"""
+    text = (f'Dear Skylar Odell, \nThank you for your 2 donations, totaling $319.52. \nFrom, Your Local Charity')
+    with open('Skylar Odell.txt', 'r+') as file:
+        fileinfo = ''
+        for line in file:
+            line.strip('\n')
+            fileinfo += line
+    assert fileinfo == text
```

Comment:
you aren't executing any of your mailroom code here - each test should execute the code you're testing.